server/api.go

```diff
@@ -201,3 +215,14 @@ func (p *Plugin) handleGetConfig(w http.ResponseWriter, r *http.Request) {
 	}
 	w.Write(data)
 }
+
+func getInstallationServiceEnvironment(installation *Installation) string {
+	if v, ok := installation.PriorityEnv[serviceEnvironmentEnvVarKey]; ok {
+		return v.Value
+	}
+	if v, ok := installation.MattermostEnv[serviceEnvironmentEnvVarKey]; ok {
+		return v.Value
+	}
+
+	return "not-defined"
```

Comment:
nit: even if one isn't defined, the server has it's own fallback. Might be slightly unclear if we assert "not-defined" when it's just defined elsewhere.

Reply:
I was wondering about this. Am I correct that `production` the default for our images?

Reply:
Yes, that's correct: it will default to `production` for anything we produce out of our CI.